os_groups.go

```diff
@@ -42,25 +48,36 @@ func (osGroups *OSGroups) ReservedArtifacts() []string {
 func (osGroups *OSGroups) RequiredScopes() scopes.Required {
 	requiredScopes := make([]string, len(osGroups.Task.Payload.OSGroups), len(osGroups.Task.Payload.OSGroups))
 	for i, osGroup := range osGroups.Task.Payload.OSGroups {
-		requiredScopes[i] = "generic-worker:os-group:" + osGroup
+		requiredScopes[i] = "generic-worker:os-group:" + config.ProvisionerID + "/" + config.WorkerType + "/" + osGroup
```

Comment:
nice change, specifying by worker type seems like the right thing to do here for sure.

Reply:
Thanks. 😄